webpages/settings/style.css

```diff
@@ -144,30 +144,23 @@ h1 {
 }
 
 .clear-button {
-  border-radius: 0 4px 4px 0px;
-  border-left: none !important;
+  border-start-radius: 0;
```

Comment:
I think this line can be removed. CSS property is invalid and all 4 corner radiuses are set below

Reply:
Removed.